### ISSUE TITLE
Scrolling carousel bug

### ISSUE THREAD

**Issue by augustfr** [tomasg88/mirage-gallery#70]
When I scroll on my trackpad, it'll sometimes move twice instead of just once. Not sure why this is happening. Doesn't feel as natural. 

We should probably have it only move once per swipe.

**Comment by santuan** [comment]
Yeah! is kind on a known issue with Swiper (the carousel we use). Specially on MacOs.

https://github.com/nolimits4web/swiper/issues/1810
https://github.com/nolimits4web/swiper/issues/1791

I tried some configurations in the `HomeHero.tsx` but was to get the horizontal scroll working with trackpad.
https://github.com/tomasg88/mirage-gallery/blob/main/components/Home/HomeHero.tsx#L31-L35

```
mousewheel={{
  forceToAxis: true,
  sensitivity: 0.01,
  thresholdDelta: 10,
  releaseOnEdges: true,
}}
```

https://github.com/nolimits4web/swiper/issues/4442

Maybe changing one of those solves the problem.

If that doesn't do the trick we could consider just for the HomeHero (that's so important because is the first thing that a user "see") using one that has that thing solve.

And only use Swiper for some "less important" carousels.

**Comment by santuan** [comment]
If you by any chance now a website that has this thing solved (as you like how it feels), we could also try to see what plugin there are using.

As far as I know swiper is best free open source for carousel.

**Comment by augustfr** [comment]
Just saw an ad for this site which has a pretty nice slider: https://www.amberjack.shop/collections/all 

Maybe we'd want something like that but then one that also has the arrows that can be clicked as well instead of the slider bar? 

Not sure.

**Comment by santuan** [comment]
That one it appear's to be using https://swiffyslider.com/

Never use it. Gonna try it and let you know :)

**Comment by santuan** [comment]
Nothing is better than the Apple way, right?

I investigate a little and end up finding this demo
https://twitter.com/Steve8708/status/1531388545960583168

It recreates the slider that Nike and Apple use. Is more native, and uses pure html and css. And a little bit of js for the arrows.
https://www.apple.com/store

You can try a demo at this link. 
https://mirage-gallery-dbefc7hpc-tomasg88.vercel.app/

It's basically the common native scroll (the scroll is hidden) and css snap
https://tailwindcss.com/docs/scroll-snap-type#horizontal-scroll-snapping

I still need to match the design and there are a few new issues with the arrows (they are skipping an image on mobile, so I remove it for now). That's because i'm not an expert in JS and this thing involves mathematics and dev skills that I don't currently know.

I found this tutorial if anyone needs more info about how it's done.
https://edgarlr.com/articles/react-carousel-using-hooks-and-css

But the scrolling should feel very smooth and also more performant.

(I also try swiffyslider and couldn't make it work on Next.js).

**Comment by augustfr** [comment]
Yeah that demo is looking pretty good. We'd probably want a bit of a taller aspect ratio so we don't crop the images so much but that slider feels way smoother.

**Comment by santuan** [comment]
New deploy :)
https://mirage-gallery-94r6uqqfq-tomasg88.vercel.app/
